core/src/epicli/data/common/ansible/playbooks/prometheus.yml

```diff
@@ -1,11 +1,16 @@
 ---
+# Ansible playbook that makes sure the base items for all nodes are installed
 
 - hosts: all
   gather_facts: yes
   tasks: [ ]  
-
+  
 - hosts: prometheus
   become: true
   become_method: sudo
   roles:
     - prometheus
+  post_tasks:
```

Comment:
I would move this to roles/prometheus/tasks/configure-k8s-apps-monitoring.yml where you have creating kubeconfig already added OR move the creation to pre_tasks. The goal is to have both in the same place.